yoti-sdk-api/src/test/java/com/yoti/api/client/docs/session/retrieve/GetSessionResultCheckTest.java

```diff
@@ -17,7 +17,7 @@
 @RunWith(MockitoJUnitRunner.class)
 public class GetSessionResultCheckTest {
 
-    private static final int SESSION_CHECKS = 10;
+    private static final int SESSION_CHECKS = 12;
```

Comment:
Shouldn't this be 13 if we're adding 3 new checks?...

Reply:
Thank you. Changed